### ISSUE TITLE
Consistency with the INSTALL documentation

### ISSUE THREAD

**Issue by Rafiot** [MISP/ansible#1]
I had a short look, would it be possible to install by default in `/var/www/MISP` ? As it is the default in INSTALL, that would be better. Or at least to pass it as a parameter.

**Comment by deloittem** [comment]
You are right, the path should become a parameter. The idea is to add Apache installation and the path should automatically be updated according to the webserver.
e.g. 
- Nginx: /usr/share/nginx/www/MISP
- Apache: /var/www/MISP
